frontend/ui/uimanager.lua

```diff
@@ -714,7 +707,9 @@ function UIManager:handleInput()
 
     -- delegate input_event to handler
     if input_event then
-        self:_resetAutoSuspendTimer()
+        if input_event.handler ~= "onInputError" then
+            self:broadcastEvent(Event:new("InputEvent"))
```

Comment:
Yeah, that's a very good point. It's an art to design API that's both efficient and ergonomic :)